### DISCUSSION TITLE
Feature: filter branches based on a prefix when using GitHub storage mode

### DISCUSSION THREAD

**Discussion by JPrisk** [Thinkmill/keystatic#455] [Ideas]
Issue: working on a project where there are lots of branches (and there are expected to be many more) - currently it is hard for content people to differentiate between their branches and general development stuff.

Suggestion: provide a `GitHubConfig` option as handler to filter branches

Examples:
`feature/content...` if wanting to isolate development/content PRs
`feature/[project-prefix]` when working in a monorepo

**Comment by JedWatson** [comment]
This seems like a great idea. We should take it into account when creating branches through the Keystatic UI as well, so that when you create a new branch from within Keystatic, it shows up.
